### PR TITLE
Feat: page preview pdf

### PR DESCRIPTION
**Pull request by rayfajars**
## Related
https://app.clickup.com/t/860rpk7aq

## Feature
- add export pdf file
- add page review pdf
- add condition if go to page review pdf not login

## Preview

Review PDF
image

Auth
image

## Evidence
title: page preview pdf
project: Jabar Super Apps
participants: @rayfajars @marsellavaleria19 @doohanas @Ibwedagama

### CODE REVIEW

File: components/Tahura/LaporanPendapatan/index.vue [hunk 1]
```diff
@@ -365,10 +369,13 @@ export default {
       this.$fetch()
     },
     clearDateRangeHandle () {
+      console.log('clear data')
```
Comment (Ibwedagama) [inline components/Tahura/LaporanPendapatan/index.vue:372]: hello @rayfajars please remove console logs

File: components/Tahura/LaporanPendapatan/index.vue [hunk 2]
```diff
@@ -466,4 +471,4 @@ export default {
 .jds-input-text {
   @apply border-gray-300 !important;
 }
-</style>
+</style>
```
Comment (Ibwedagama) [inline components/Tahura/LaporanPendapatan/index.vue:474]: please add empty line below this closing tag @rayfajars

File: pages/tahura/preview-pdf/_assurance-category-startDate-endDate-status.vue [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+        total += category.quantity * this.checkAsurance(category.price)
+      })
+      return total || '-'
+    },
+    async generatePdf () {
+      try {
+        await this.$axios.post('/ticket/tahura/income/report', {
+          url: window.location.href
+        })
+        console.log('generate pdf success')
```
Comment (Ibwedagama) [inline pages/tahura/preview-pdf/_assurance-category-startDate-endDate-status.vue:303]: please remove this console as well @rayfajars

**Review by Ibwedagama — APPROVED** [review]

File: pages/tahura/preview-pdf/_assurance-category-startDate-endDate-status.vue (hunk 3 above)
Comment (rayfajars) [reply pages/tahura/preview-pdf/_assurance-category-startDate-endDate-status.vue]: i change [here](https://github.com/jabardigitalservice/super-app-cms/pull/124/commits/35a838545f31f6c04b40e11e98a703b4d897a3d6)

File: components/Tahura/LaporanPendapatan/index.vue (hunk 2 above)
Comment (rayfajars) [reply components/Tahura/LaporanPendapatan/index.vue:473]: i am change [here](https://github.com/jabardigitalservice/super-app-cms/pull/124/commits/b618d3ee0bf307c5cdafb2b1bb40572be69ac6ae)

File: components/Tahura/LaporanPendapatan/index.vue (hunk 1 above)
Comment (rayfajars) [reply components/Tahura/LaporanPendapatan/index.vue]: i am change [here](https://github.com/jabardigitalservice/super-app-cms/pull/124/commits/b618d3ee0bf307c5cdafb2b1bb40572be69ac6ae)

**Review by doohanas — APPROVED** [review]

**Review by marsellavaleria19 — APPROVED** [review]

**Pull request merged**